FinSearchUnified/Resources/views/frontend/listing/filter/_includes/filter-color-selection.tpl

```diff
@@ -21,9 +21,13 @@
     <label class="filter-panel--label"
            for="__{$facet->getFieldName()|escape:'htmlall'}__{$option->getId()|escape:'htmlall'}">
 
-        {if $option->getColorcode()}
-            {$mediaFile = $option->getColorcode()}
-            <div class="color--filter-item" style="background: {$mediaFile};" title="{$option->getId()|escape:'htmlall'}"></div>
+        {$colorCode = $option->getColorcode()}
+        {$imageUrl = $option->getImageUrl()}
+
+        {if $colorCode && $imageUrl}
```

Comment:
Shouldn't checking for `$imageUrl` be sufficient? We are we also checking for `$colorCode`?

Reply:
I added a check for `$imageUrl` separately, so we can provide the no-result image as backup